homeassistant/components/asuswrt/strings.json

```diff
@@ -2,7 +2,7 @@
   "config": {
     "step": {
       "user": {
-        "title": "AsusWRT",
+        "title": "ASUSWRT connection settings",
```

Comment:
Titles are implied / have a default for this. This field can actually just be omitted:

```suggestion
```